package.json

```diff
@@ -19,7 +18,6 @@
     "coverage": "COVERAGE=true npm run unit",
     "node6": "node utils/node6-transform/index.js",
     "test-node6": "jasmine utils/node6-transform/test/test.js",
-    "build": "npm run node6",
```

Comment:
I think this was unintentional removal.

Reply:
That's a drive-by. Do we need it?

Reply:
I wanted to future proof, in case we have more build steps.